### ISSUE TITLE
Login dialog text doesn't read by screenreader (to musescore.com/audio.com)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#19477]
### Issue type

Accessibility

### Bug description

![Capture](https://github.com/musescore/MuseScore/assets/90187801/2e98d15f-5d8d-427a-8ad9-e7ed33d840e0)

### Steps to reproduce

1. Turn on screenreader
2. Go to File>Publish to MuseScore.com or File>Share on Audio.com (you should be not signed in)

### Screenshots/Screen recordings

https://github.com/musescore/MuseScore/assets/90187801/7db70ccc-8861-4f91-bf6a-c41355534208

### MuseScore Version

4.2.0 master

### Regression

No.

### Operating system

all

**Comment by mathesoncalum** [comment]
@DmitryArefiev this seems to be working fine on my end (macOS) - I suspect this may have been fixed in #19429
